config/keys.js

```diff
@@ -17,3 +17,8 @@ module.exports = {
     secret: process.env.JWT_SECRET
   }
 };
+
+// Previous mongodb setting was:
+// mongodb: {
+//   dbURI: `mongodb://${process.env.MONGO_USERNAME}:${process.env.MONGO_PASSWORD}@${process.env.MONGO_HOSTNAME}:${process.env.MONGO_PORT}/${process.env.MONGO_DB}?authSource=admin`
+// }
```

Comment:
I think we can remove the commented out code.

Reply:
commented code was deleted